rustls/src/client/client_conn.rs

```diff
@@ -267,12 +267,12 @@ impl ClientConfig {
 /// # let _: ServerName = x;
 /// ```
 #[non_exhaustive]
-#[derive(Clone, Debug, Eq, Hash, PartialEq)]
+#[derive(Clone, Eq, Hash, PartialEq)]
 pub enum ServerName {
     /// The server is identified by a DNS name.  The name
     /// is sent in the TLS Server Name Indication (SNI)
     /// extension.
-    DnsName(verify::DnsName),
+    DnsName(webpki::DnsName),
```

Comment:
The reason we don't want this is that this exposes the `webpki` crate as part of the public API, which means that the public API for webpki and rustls have to evolve in lockstep (in particular, rustls cannot start using a semver-incompatible new version like 0.101 without also bumping our own version in a semver-incompatible way). Therefore, I don't think we want to make this change.